### ISSUE TITLE
[Feature] Subtask in Timeline display Proyect Name like in Normal Task (not only colored circle)

### ISSUE THREAD

**Issue by serhumanos** [johannesjo/super-productivity#2314]
Actually subtask display a circle with the color of the associated project, that is very cool, but I think it wil be much better if the name of the project were displayed like a normal task (next to the maybe comming soon "subtask tags" (#2234))

**Comment by serhumanos** [comment]
> This issue has not received any updates in 90 days. Please comment, if this still relevant!

Hi! for me this is still relevant when I use the timeline view.. for now I have to use the Project View to focus better.

**Comment by timmillard** [comment]
I think I get what you mean, but could you go into a bit more detail?

**Comment by serhumanos** [comment]
> I think I get what you mean, but could you go into a bit more detail?

sure, actually in timeline view, the first icon (colored dot) represent the project of the task, followed by the task name. So it will be great that the name of the project will appear next to the colored dot. In today view, the name of the project appears in the task header.
I hope that the idea was clear
thanks!

**Comment by timmillard** [comment]
So you would like the project name to be in the header, and the task name to be 'secondary' to that, like a subtitle underneath it? See this image for what I think you want: 

![image](https://user-images.githubusercontent.com/115703829/233965309-b4362a39-52b8-4105-b0b9-129d4fc85692.png)

**Comment by serhumanos** [comment]
The issue is in Timeline view.. with SubTask..

in the following image, it shows 3 subtask (2 from project 1 and 1 from proyect 2)
![imagen](https://user-images.githubusercontent.com/7744939/234417054-92bff20c-b7b2-4f53-a210-b59729278277.png)

when I change to Timeline view the project name of the subtask are hidden, as is shown in the following image
![imagen](https://user-images.githubusercontent.com/7744939/234418043-df579664-85d9-4029-88e5-286aa5fc49f1.png)

My idea is show Project, Task and subtask, something like this:
![imagen](https://user-images.githubusercontent.com/7744939/234419195-ee37fce0-0bd2-4b99-bf38-9ba10d247ec7.png)

**Comment by johannesjo** [comment]
Thanks for the suggestion! I generally like the idea of having the parent task title. It gets a bit problematic for longer task titles though and it also raises some ui problems for the edit task input, since this is now positioned differently and it raises further problems with the tags and project, since those are not editable for a sub task (only by changing the parent).  Since I haven't solved those problems, I went with what we got now...
